package.json

```diff
@@ -34,7 +34,7 @@
     "webpack": "^1.4.13"
   },
   "peerDependencies": {
-    "react": "0.12.x"
+    "react": ">=0.12.0"
```

Comment:
Are you sure this is going to work on `0.14` when it gets released? Using the [semver calculator](http://semver.npmjs.com/) `^0.12 || ^0.13` will match 0.12 and 0.13.